src/ja/server/diagrams/cancelLowLevel.puml

```diff
@@ -0,0 +1,55 @@
+
+@startuml
+
+box "Central Server" #Lightblue
+    participant ":CancelActivity"
+    participant ":Scheduler"
+    participant ":Authenticator"
+    participant ":DataBaseHandler"
+    participant ":Dispatcher"   
+end box
+
+
+-> ":CancelActivity":
+activate ":CancelActivity"
+
+":CancelActivity" -> ":Scheduler" ++: cancelJob(jobID:int, userID:int)
```

Comment:
The Scheduler should not be invoked to cancel a Job.
Cancel requests should be sent directly to the dispatcher.
The Scheduler should be invoked afterwards to re-evaluate which Jobs should be run on which machine.